### PR TITLE
Update minikube and community pixie

### PR DESCRIPTION
**Pull request by elaguerta**
Make some changes to remove friction points during setup. I encountered these while getting set up for development, but they could help all users. 

- In the minikube setup doc, update the `Verify` section. Delete the `minikube dashboard` command because that doesn't seem to provide info. Keep `kubectl get nodes`. Add `kubectl config current-context` to verify that k8 is pointing to the correct cluster.
- In the pixie community install doc, add a section for deleting pixie and the namespace in case of a failed `px deploy`.

### CODE REVIEW

**Review by zasgar — DISMISSED** [review]

File: content/en/02-installing-pixie/03-install-guides/01-community-cloud-for-pixie.md [hunk 1]
```diff
@@ -76,6 +76,13 @@ px-operator         vizier-operator
 
 To deploy Pixie to another cluster, change your `kubectl config current-context` to point to that cluster. Then repeat the same deploy commands shown in this step.
 
+If the `px deploy` does not successfully complete you might retry it after deleting all the nodes and the namespace as sometimes the deploy nodes can get into a bad state:
```
Comment (hmstepanek) [inline content/en/02-installing-pixie/03-install-guides/01-community-cloud-for-pixie.md]: @nserrino said this issue would be fixed in the next release [here](https://github.com/pixie-io/pixie/pull/324#issuecomment-931614812) so I think we should just leave this out of the docs.

File: content/en/02-installing-pixie/02-setting-up-k8s/01-minikube-setup.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ To deploy a specific K8s version [supported by Pixie](/installing-pixie/requirem
 
 ## Verify
 
-Run `minikube dashboard` or `kubectl get nodes` to verify your cluster is up and running.
+Run `kubectl get nodes` to verify your cluster is up and running. Run 'kubectl config current-context` to verify that K8s is pointing to your cluster.
```
Comment (hmstepanek) [inline content/en/02-installing-pixie/02-setting-up-k8s/01-minikube-setup.md]: I know k8 is an abbreviation for kubernetes but I think it's probably better to just be explicit here and say `the Kubernetes command-line tool` or `kubectl`.

**Review by hmstepanek — CHANGES_REQUESTED** [review]
Thanks for adding these! I'm super excited that we can contribute to improving the docs for future developers! Had a couple small bits of feedback.

File: content/en/02-installing-pixie/03-install-guides/01-community-cloud-for-pixie.md (hunk 1 above)
Comment (elaguerta) [reply content/en/02-installing-pixie/03-install-guides/01-community-cloud-for-pixie.md]: I've removed it, thanks!

File: content/en/02-installing-pixie/02-setting-up-k8s/01-minikube-setup.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ To deploy a specific K8s version [supported by Pixie](/installing-pixie/requirem
 
 ## Verify
 
-Run `kubectl get nodes` to verify your cluster is up and running. Run 'kubectl config current-context` to verify that K8s is pointing to your cluster. 
+Run `kubectl get nodes` to verify your cluster is up and running. Run 'kubectl config current-context` to verify that `kubectl` is pointing to your cluster.
```
Comment (htroisi) [inline content/en/02-installing-pixie/02-setting-up-k8s/01-minikube-setup.md]:
```suggestion
Run `kubectl get nodes` to verify your cluster is up and running. Run `kubectl config current-context` to verify that `kubectl` is pointing to your cluster. 
```

Comment (elaguerta) [reply content/en/02-installing-pixie/02-setting-up-k8s/01-minikube-setup.md]: Thanks for the catch!

**Review by nserrino — APPROVED** [review]

**Pull request merged**